src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -81,6 +81,11 @@ export const ProjectCard = ({
   isDragging,
   ...other
 }: Props) => {
+  if (number && !numberAriaLabel && process.env.NODE_ENV !== 'production') {
+    console.warn(
```

Comment:
Should it be an error since it's an accessibility bug to not have an aria label on this number?

Reply:
Yuh will change to error, thanks!